ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -144,7 +202,7 @@ export default defineComponent({
       lengthTrust.trustIndenture = val
       setLengthTrust(lengthTrust)
     })
-    watch(() => localState.lifeInfinite, (val: boolean) => {
+    watch(() => localState.lifeInfinite, (val: string) => {
```

Comment:
why change to a string here?

Reply:
The radio buttons are actually a string. As a string the initial state of both deselected can be represented as ''.